### PR TITLE
HTS: Implement NFT Wipe (+ refactor Fungible wipe)

### PR DESCRIPTION
**Pull request by joan41868**
**Description**:

This PR modifies the `Token` class in order to add wipe functionality for unique/common tokens.
As this PR provides functionality for `common` tokens, #1672 may be obsolete now.
**Related issue(s)**:

Fixes #
Burn / Wipe fee usage calc.
Merge conflicts with `master` branch.
**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by Daniel-K-Ivanov — DISMISSED** [review]
LGTM

**Review by tinker-michaelj — DISMISSED** [review]
LGTM

File: test-clients/src/main/java/com/hedera/services/bdd/spec/HapiSpecOperation.java [hunk 1]
```diff
@@ -206,6 +206,7 @@ private AccountID randomNodeFrom(HapiApiSpec spec) {
 				updateStateOf(spec);
 			}
 		} catch (Throwable t) {
+			t.printStackTrace();
```
Comment (tinker-michaelj) [inline test-clients/src/main/java/com/hedera/services/bdd/spec/HapiSpecOperation.java:209]: Even if this is very helpful for local debugging, our regression does a lot of log parsing and fails on unexpected `ERROR`-type message, so this unconditional stack trace is killer. 😁

**Review by Daniel-K-Ivanov — APPROVED** [review]
LGTM

**Comment by tinker-michaelj** [comment]
Merged via #1755

**Review by qnswirlds — COMMENTED** [review]
More changes in #1762

**Pull request closed without merging**